### PR TITLE
[TGER-60] - remove guarded property

### LINKED ISSUE #37: Review $guarded from Reviews package
Update composer.json also

### PR DESCRIPTION
**Pull request by arwaawan3**
Closes #37

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "require": {
         "php": "^7.4|^8.0",
         "google/apiclient": "^2.7",
-        "tipoff/support": "^1.1.2"
+        "tipoff/support": "^1.2.6"
     },
     "require-dev": {
-        "tipoff/test-support": "^1.0"
+        "tipoff/test-support": "^1.0.3"
```
Comment (drewroberts) [inline composer.json:23]: Thanks for bumping to the current releases!

**Comment by drewroberts** [comment]
@phuclh Can you issue a new PR on this package to fix the psalm errors in the models?

File: composer.json (hunk 1 above)
Comment (arwaawan3) [reply composer.json:23]: 👍  Sorry I didn't know about it before

**Review by drewroberts — APPROVED** [review]
Thank you, Arwa! 🚁

**Review by joshtorres — COMMENTED** [review]
Looks good!

**Pull request merged**